### ISSUE TITLE
Can't get virtualenvs working (PATCH has no effect)

### ISSUE THREAD

**Issue by rayterrill** [ansible/awx#5291]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
Can't get virtualenvs picked up by the API. No errors reported, but also doesn't appear to do anything (GET on /api/v2/settings/system returns no changes). Confirmed both containers contain the custom virtualenv

##### ENVIRONMENT
* AWX version: 9.0.1.0
* AWX install method: docker on linux
* Ansible version:  2.8.5
* Operating System: Ubuntu
* Web Browser: Chrome/Firefox

##### STEPS TO REPRODUCE

1. Modified awx/installer/inventory to uncomment line:
```
custom_venv_dir=/opt/my-envs/
```
2. Rebuilt containers:
```
ansible-playbook -i inventory install.yml
```
3. Attempted to issue PATCH command against AWX API. No errors reported:
```
curl --insecure -v -X PATCH 'https://admin:password@awx.ourdomain.com/api/v2/settings/system' -d '{"CUSTOM_VENV_PATHS": ["/opt/my-envs/custom-venv"]}' -H 'Content-Type: application/json'

*   Trying 172.31.29.13...
* TCP_NODELAY set
* Connected to awx.ourdomain.com (172.31.29.13) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
* (304) (OUT), TLS handshake, Client hello (1):
* (304) (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-RSA-CHACHA20-POLY1305
* ALPN, server accepted to use h2
* Server certificate:
*  subject: C=US; ST=Oregon; L=Portland; O=Our Company; OU=IT; CN=*.ourdomain.com
*  start date: Jul 30 00:00:00 2018 GMT
*  expire date: Nov  1 00:00:00 2020 GMT
*  issuer: C=US; O=DigiCert Inc; CN=DigiCert SHA2 Secure Server CA
*  SSL certificate verify result: unable to get local issuer certificate (20), continuing anyway.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Server auth using Basic with user 'admin'
* Using Stream ID: 1 (easy handle 0x55f16107e580)
> PATCH /api/v2/settings/system HTTP/2
> Host: awx.ourdomain.com
> Authorization: Basic <REMOVED>
> User-Agent: curl/7.58.0
> Accept: */*
> Content-Type: application/json
> Content-Length: 51
> 
* Connection state changed (MAX_CONCURRENT_STREAMS updated)!
* We are completely uploaded and fine
< HTTP/2 301 
< server: nginx/1.14.0 (Ubuntu)
< date: Mon, 11 Nov 2019 15:41:39 GMT
< content-type: text/html; charset=utf-8
< content-length: 0
< location: /api/v2/settings/system/
< vary: Accept-Language, Origin, Cookie
< content-language: en
< x-api-total-time: 0.020s
< strict-transport-security: max-age=15768000
< content-security-policy: default-src 'self'; connect-src 'self' ws: wss:; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.pendo.io; img-src 'self' *.pendo.io data:; report-uri /csp-violation/
< x-content-security-policy: default-src 'self'; connect-src 'self' ws: wss:; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.pendo.io; img-src 'self' *.pendo.io data:; report-uri /csp-violation/
< x-frame-options: DENY
< 
* Connection #0 to host awx.ourdomain.com left intact
```
4. Verified via GET request that the CUSTOM_VENV_PATHS is still blank:
```
{
    "ACTIVITY_STREAM_ENABLED": true,
    "ACTIVITY_STREAM_ENABLED_FOR_INVENTORY_SYNC": false,
    "ORG_ADMINS_CAN_SEE_ALL_USERS": true,
    "MANAGE_ORGANIZATION_AUTH": true,
    "TOWER_URL_BASE": "https://towerhost",
    "REMOTE_HOST_HEADERS": [
        "REMOTE_ADDR",
        "REMOTE_HOST"
    ],
    "PROXY_IP_WHITELIST": [],
    "LICENSE": {},
    "REDHAT_USERNAME": "",
    "REDHAT_PASSWORD": "",
    "AUTOMATION_ANALYTICS_URL": "https://example.com",
    "INSTALL_UUID": "8122a8bb-4a83-4f02-8840-3592e5a9efb3",
    "CUSTOM_VENV_PATHS": [],
    "INSIGHTS_TRACKING_STATE": false,
    "BROKER_DURABILITY": true
}
```

##### EXPECTED RESULTS

I would expect an error if something was wrong, or the CUSTOM_VENV_PATHS to be updated.

##### ACTUAL RESULTS

Nothing.

**Comment by rayterrill** [comment]
I tracked this down - this seems like a documentation bug perhaps?

I tracked down this doc (https://docs.ansible.com/ansible-tower/latest/html/administration/tipsandtricks.html), which shows the custom virtual env configuration, and the URL for the API has a trailing slash "/api/v2/settings/system/". The AWX documentation does not (https://github.com/ansible/awx/blob/devel/docs/custom_virtualenvs.md).

As soon as I added the trailing slash, I got the error that the directory didn't exist. Modified the inventory file to uncomment the custom_venv_dir config, rebuilt my containers, and I'm up and running.
